sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -426,6 +426,36 @@ public enum ServiceBusTransportType
         AmqpWebSockets = 1,
     }
 }
+namespace Azure.Messaging.ServiceBus.Amqp
+{
+    public enum AmqpBodyType
+    {
+        Unspecified = 0,
+        Data = 1,
+        Sequence = 2,
+        Value = 3,
+    }
+    public static partial class ServiceBusMessageExtensions
+    {
+        public static Azure.Messaging.ServiceBus.Amqp.AmqpBodyType GetAmqpBodyType(this Azure.Messaging.ServiceBus.ServiceBusMessage message) { throw null; }
+        public static System.Collections.Generic.IEnumerable<System.ReadOnlyMemory<byte>> GetAmqpDataBody(this Azure.Messaging.ServiceBus.ServiceBusMessage message) { throw null; }
+        public static System.Collections.Generic.IEnumerable<System.Collections.IList> GetAmqpSequenceBody(this Azure.Messaging.ServiceBus.ServiceBusMessage message) { throw null; }
+        public static object GetAmqpValueBody(this Azure.Messaging.ServiceBus.ServiceBusMessage message) { throw null; }
+    }
+    public static partial class ServiceBusReceivedMessageExtensions
+    {
+        public static Azure.Messaging.ServiceBus.Amqp.AmqpBodyType GetAmqpBodyType(this Azure.Messaging.ServiceBus.ServiceBusReceivedMessage message) { throw null; }
+        public static System.Collections.Generic.IEnumerable<System.ReadOnlyMemory<byte>> GetAmqpDataBody(this Azure.Messaging.ServiceBus.ServiceBusReceivedMessage message) { throw null; }
+        public static System.Collections.Generic.IEnumerable<System.Collections.IList> GetAmqpSequenceBody(this Azure.Messaging.ServiceBus.ServiceBusReceivedMessage message) { throw null; }
```

Comment:
We don't use non generic collection in .net APIs. Untyped lists should be IList<object>. Having said that, I need to understand this whole feature. I wonder if we can avoid untyped APIs altogether.